src/transforms/ampDisableCacheTransform.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const AmpOptimizer = require('@ampproject/toolbox-optimizer');
+
+/**
+ * Removes the `amp` attribute from the `html` tag to disable serving
+ * the page from the AMP Cache. We do this in a separate transformer
+ * to still support AMP validation.
+ */
+const ampDisableCacheTransform = (eleventyConfig, options = {}) => {
+  if (options.ampCache !== false) {
```

Comment:
any reason why this isn't just explinctly checking for true? Are you wanting 

```
{
  "ampCache": 42,
  "ampCache": {},
  "ampCache": "false",
}
``` 

to work?

Reply:
The default value is `undefined` which I want to correspond to `true`.

Reply:
So yes, this is WAI.